### PR TITLE
Vet center pages css fixes

### PR DESCRIPTION
**Pull request by brangi**
resolves - https://github.com/department-of-veterans-affairs/va.gov-team/issues/22728

### CODE REVIEW

File: src/applications/static-pages/sass/modules/_m-facilities.scss [hunk 1]
```diff
@@ -364,4 +364,10 @@ div.field-vet-center-feature-content  {
   li {
     margin-bottom: 0px
   }
+}
+
+.h4-vet-center {
+  font-size: 16px;
+  font-family: "Source Sans Pro";
+  font-weight: 700;
```
Comment (mpelzsherman) [inline src/applications/static-pages/sass/modules/_m-facilities.scss:372]: Can we use `force-small-header` here instead?

We can move it outside of `facilities_health_service` if need be.

Comment (brangi) [reply src/applications/static-pages/sass/modules/_m-facilities.scss:372]: yeah, didn't realize values are the same

**Comment by mpelzsherman** [comment]
I'm seeing the following issues on [Tugboat](https://web-zxlmoigkwacrdik2fa9rcshntpmrauhr.demo.cms.va.gov/escanaba-vet-center/):

- H4 font family should be "Source Sans Pro"
- 'Back to top' is not fixed to the right side of viewport. @lildziner said we can leave this as is since this is consistent with other pages.
- Images are 250 x 150. Need to confirm with @lildziner 
- Spacing between locations is ~55px; should be 40px
- Alerts overlap photos at 481 - 545px:
__Veterans_Affairs
- Spacing between locations at < 480px is too small, should be 32px

File: src/applications/static-pages/sass/modules/_m-facilities.scss [hunk 2]
```diff
@@ -364,6 +364,12 @@ div.field-vet-center-feature-content  {
   li {
     margin-bottom: 0px
   }
+  ul {
+    margin-top: 0px
+  }
```
Comment (mpelzsherman) [inline src/applications/static-pages/sass/modules/_m-facilities.scss:369]: We should probably remove the margin-top for `ol` as well.

**Comment by brangi** [comment]
> Please make sure this doesn't break VAMC pages. Check e.g. https://www.va.gov/pittsburgh-health-care/locations/pittsburgh-va-medical-center-university-drive/
> 
> VA_Pittsburgh_Health_Care___Pittsburgh_VA_Medical_Center-University_Dr

will do

**Comment by brangi** [comment]
I will close this PR in order to move liquid changes to the content build

**Pull request closed without merging**